permissions/event-model.https.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Permissions API - Event Model Tests</title>
+<link rel="help" href="https://www.w3.org/TR/permissions/" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<body>
+<script>
+'use strict';
+
+/**
+ * 1. Multiple listeners on one PermissionStatus
+ */
+promise_test(async t => {
+  // Start from "prompt"
+  await test_driver.set_permission({ name: 'geolocation' }, 'prompt');
+  const status = await navigator.permissions.query({ name: 'geolocation' });
+  assert_equals(status.state, 'prompt', 'Initial state is "prompt"');
+
+  // We'll create two promises, one for each listener
+  const promise1 = new Promise(resolve => {
+    status.addEventListener('change', () => resolve('addEventListener-fired'));
+  });
+  const promise2 = new Promise(resolve => {
+    status.onchange = () => resolve('.onchange-fired');
+  });
+
+  // Trigger a single transition -> "granted"
+  await test_driver.set_permission({ name: 'geolocation' }, 'granted');
+
+  // Wait for both listeners
+  const results = await Promise.all([promise1, promise2]);
+  results.sort();
+
+  assert_equals(status.state, 'granted', 'State should now be "granted"');
+  assert_array_equals(
+    results,
+    ['.onchange-fired', 'addEventListener-fired'].sort(),
```

Comment:
nit: why does `.onchange-fired` start with `.` but `addEventListener-fired` doesn't?

Reply:
Oh, is it a sort hack?